lib/clubs.jsx

```diff
@@ -0,0 +1,116 @@
+var React = require('react');
+var Router = require('react-router');
+var Link = Router.Link;
+
+var Page = require('./page.jsx');
+var HeroUnit = require('./hero-unit.jsx');
+var Blockquote = require('./blockquote.jsx');
+var IconLink = require('./icon-link.jsx');
+var PageEndCTA = require('./page-end-cta.jsx');
+
+var WebLitMap = React.createClass({
+  render: function() {
+    return(
+      <div className="row">
+        <div className="col-sm-12 col-md-12 col-lg-12">
+          <h2>The Global Web Literacy Movement</h2>
+          <div>placeholder for map</div>
+        </div>
+      </div>
+    );
+  }
+});
+
+var HowClubWorks = React.createClass({
+  render: function() {
+    return(
+      <div className="row">
+        <div className="col-sm-4 col-md-4 col-lg-4">
+          <img src="/img/icon-how-do-clubs-work.svg" alt="icon how do clubs work" />
+        </div>
+        <div className="col-sm-8 col-md-8 col-lg-8">
+          <h2>How do Clubs work?</h2>
+          <ul>
+            <li>Grow the web literary of leaners</li>
+            <li>Meet regularly in classrooms, libraries, coffee shops &mdash; anywhere!</li>
+            <li>Teach with open practices</li>
+            <li>Guide people to learn by making</li>
+            <li>Connect with local and global networks</li>
+          </ul>
+        </div>
+      </div>
+    );
+  }
+});
+
+var IconLinks = React.createClass({
+  ICON_LINKS: [
+    {
+      linkTo: "/fixme",
+      imgSrc: "/img/icon-curriculum.svg",
+      imgAlt: "icon curriculum",
+      head: "Curriculum",
+      subhead: "Modular Web Literacy curriculum"
+    },
+    {
+      linkTo: "/fixme",
+      imgSrc: "/img/icon-connect.svg",
+      imgAlt: "icon connect",
+      head: "Connect",
+      subhead: "Connect with other Club Leaders"
+    },
+    {
+      linkTo: "/fixme",
+      imgSrc: "/img/icon-tips.svg",
+      imgAlt: "icon tips",
+      head: "Helpful Tips",
+      subhead: "Tips for running your Club"
+    }
+  ],
+  render: function() {
+    return(
+      <div className="row">
+        <div className="col-sm-4 col-md-4 col-lg-4">
+          <IconLink info={this.ICON_LINKS[0]} />
```

Comment:
Oh yea, I think in this case we can loop it through since each of the icon link has the same width.  Cool, I'll change what's on the `homepage` as well.